app/models/skin.rb

```diff
@@ -498,6 +516,7 @@ def self.create_default
     end
     skin.official = true
     skin.save!
+    Rails.cache.increment('skins_generation/'+skin.type.nil? ? ("site_skin") : skin.id.to_s )
```

Comment:
Surrounding space missing for operator `+`.<br>Space inside parentheses detected.